pettingzoo/utils/wrappers/base.py

```diff
@@ -19,72 +18,12 @@ def __init__(self, env: AECEnv[AgentID, ObsType, ActionType]):
         super().__init__()
         self.env = env
 
-        try:
-            self.possible_agents = self.env.possible_agents
-        except AttributeError:
-            pass
-
-        self.metadata = self.env.metadata
-
-        # we don't want these defined as we don't want them used before they are gotten
-
-        # self.agent_selection = self.env.agent_selection
-
-        # self.rewards = self.env.rewards
-        # self.dones = self.env.dones
-
-        # we don't want to care one way or the other whether environments have an infos or not before reset
-        try:
-            self.infos = self.env.infos
-        except AttributeError:
-            pass
-
-        # Not every environment has the .state_space attribute implemented
-        try:
-            self.state_space = (
-                self.env.state_space  # pyright: ignore[reportGeneralTypeIssues]
-            )
-        except AttributeError:
-            pass
-
     def __getattr__(self, name: str) -> Any:
         """Returns an attribute with ``name``, unless ``name`` starts with an underscore."""
-        if name.startswith("_"):
+        if name.startswith("_") and name != "_cumulative_rewards":
             raise AttributeError(f"accessing private attribute '{name}' is prohibited")
         return getattr(self.env, name)
 
-    @property
```

Comment:
So removing this is because the underlying env has the action spaces et al defined and we just inherit directly through getattr?